scipy/stats/tests/test_distributions.py

```diff
@@ -1064,6 +1064,55 @@ class TestTruncnorm:
     def setup_method(self):
         np.random.seed(1234)
 
+    @pytest.mark.parametrize("a, b, ref",
+    [(0, 100, 0.7257913526447274),
+    (0.6, 0.7, -2.3027610681852573),
+    (1e-06, 2e-06, -13.815510557964274)])
+    def test_entropy(self, a, b, ref):
+        #All reference values were calculated with mpmath:
+        #import mpmath as mp
+        #mp.dps = 50
+        #def entropy_trun(a, b):
+        #    def cdf(x):
+        #        return 0.5 * (1 + mp.erf(x / mp.sqrt(2)))
+        #
+        #    Z = cdf(b) - cdf(a)
+        #
+        #    def pdf_standard_norm(x):
+        #        return (mp.exp(-0.5 * x ** 2) / mp.sqrt(2 * mp.pi))
+        #
+        #    def pdf(x):
+        #        return pdf_standard_norm(x) / Z
+        #
+        #    return -mp.quad(lambda t: pdf(t) * mp.log(pdf(t)), [a, b])
+        assert_allclose(stats.truncnorm.entropy(a, b), ref, rtol=1e-10)
+
+    @pytest.mark.parametrize("a, b, ref",
+    [(1e-11, 10000000000.0, 0.725791352640738),
+    (1e-100, 1e+100, 0.7257913526447274),
+    (-1e-100, 1e+100, 0.7257913526447274),
+    (-1e+100, 1e+100, 1.4189385332046727),
+    (1e-11, 1.0001e-11, -34.8181418495907)])
```

Comment:
This should be indented to follow PEP8 even if CI is not catching it, since it will have to be changed eventually. The best option is aligning with the end of `parametrize(`.